pkg/backends/locust/resources.go

```diff
@@ -264,7 +265,8 @@ func newWorkerJob(
 			Template: coreV1.PodTemplateSpec{
 				ObjectMeta: metaV1.ObjectMeta{
 					Labels: map[string]string{
-						"app": name,
+						"app":            name,
```

Comment:
```suggestion
						"name":            name,
```
maybe this is what is actually wrong, just need to test to see if it works